modules/retail/pages/retail-mass-config.adoc

```diff
@@ -72,19 +85,21 @@ retail_yaml --from-yaml filename.yaml
 See the [command]``retail_yaml --help`` output for additional options.
 . In the {susemgr} {webui}, check that your systems are listed and displaying correctly, and the formulas you require are available.
 . Create activation keys for each of your branch servers, connect them using bootstrap, and configure them as proxy servers.
-// FIXME: look up exact refs
-For further information on these steps, see the {susemgr} documentation.
-. In the [guimenu]``States`` tab, click btn:[Apply Highstate] to deploy your configuration changes.
+For further information, xref:retail:retail-install-unified.adoc[].
+. In the [guimenu]``States`` tab, click btn:[Apply Highstate] to deploy your configuration changes for each branch server.
 . Connect your terminals according to your infrastructure plan.
 
 If you need to change your configuration, you can edit the YAML file at any time, and use the [command]``retail_yaml --from-yaml`` command to upload the new configuration.
 
+
+== Exporting Configuration to Mass Configuration File
```

Comment:
```suggestion
== Export Configuration to Mass Configuration File
```

Reply:
I fixed this in my changes, but I am little confused here. Because in another file, @Loquacity suggested to use present continuous tense. What tense should be used in headers?

Reply:
Use simple present for headings, and present continuous for procedures. We're a bit more specific than that, though, by looking at the verb type instead of the tenses: use bare infinitive verb form for the section heading, gerund verb form for the procedure title (cf:  https://github.com/SUSE/doc-susemanager/wiki/wug#headings and https://github.com/SUSE/doc-susemanager/wiki/wug#procedure-titles I will update this with the tense info, though, since you made me notice it's missing!). 

It's entirely possible I did the wrong thing somewhere along the line though, I *am* human :wink:

Reply:
Thanks for explanation and the links to the wiki, I completely missed that one. :)